x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/alert_actions.tsx

```diff
@@ -114,13 +85,20 @@ export function AlertActions({
     closeActionsPopover();
   };
 
-  const handleUntrackAlert = useCallback(async () => {
-    await untrackAlerts({
-      indices: [ecsData?._index ?? ''],
-      alertUuids: [alertId],
-    });
-    onSuccess();
-  }, [untrackAlerts, alertId, ecsData, onSuccess]);
+  const DefaultAlertActions = useMemo(
+    () =>
+      triggersActionsUi?.getAlertsTableDefaultAlertActions({
+        key: 'defaultRowActions',
+        onActionExecuted: closeActionsPopover,
+        isAlertDetailsEnabled: false,
```

Comment:
Why is it set to `false`? Based on the name I assumed it's for opening the alert flyout, but perhaps it means something else. There is no comment on what this prop is intended.

Reply:
You're right, it can be misleading. This comes from the check that o11y performs to decide whether to open the flyout or the page (`true` refers to opening the detail page). I'll ad a JSDoc to clarify and make a note to improve the naming asap 🙂